README.md

```diff
@@ -69,6 +69,14 @@ Consume from the queue named `test`:
 amqpcat --consumer --uri=$CLOUDAMQP_URL --queue test
 ```
 
+Consume from the queue named `test`, dumping each message into a given
+directory, instead of STDOUT, deriving the filename from the message
+properties, or timestamp if no properties are set:
+
+```sh
+amqpcat --consumer --uri=$CLOUDAMQP_URL --queue test
```

Comment:
I guess you should have `-d` in the example here

Reply:
fixed, I think I lost all my README changes to git-monster...